src/components/FilterChecklist.vue

```diff
@@ -60,12 +58,15 @@ fieldset {
   cursor: pointer;
   margin-bottom: 16px;
 }
-
 .filter-dropdown-visibility-button {
   float: right;
   cursor: pointer;
 }
-
+.rotImg{
+  transform: rotate(180deg);
+   float: right;
```

Comment:
If you use the class binding I suggested above you can keep this class style with only `transform: rotate(180deg);`

Reply:
This, and I think it would also be best to do the class binding on the img element itself instead of the button. (I mentioned this in my last review, along with Breno's request for the README change. Please be sure to read thoroughly and ask questions if you have them!)

Reply:
Actually I haven't changed the Readme file, is showing because in this PR, there is commit from another user,
So can I create a new PR for this issue? And close this one?

Reply:
Since I got warning about line length being exceed by 100, so i have used 
<!-- eslint-disable-next-line -->
@brenoferreira